hvplot/tests/testgeo.py

```diff
@@ -480,3 +480,75 @@ def test_proj_to_cartopy_wkt_string(self):
         assert isinstance(crs, self.ccrs.RotatedPole)
         assert crs.proj4_params['lon_0'] == 357.5
         assert crs.proj4_params['o_lat_p'] == 37.5
+
+    def test_proj_to_cartopy_albers_equal_area_wkt(self):
+        """Test Albers Equal Area projection from WKT string"""
+        from ..util import proj_to_cartopy
+
+        albers_wkt = 'PROJCS["Projection = Albers Conical Equal Area",GEOGCS["WGS 84",DATUM["WGS_1984",SPHEROID["WGS 84",6378137,298.257223563,AUTHORITY["EPSG","7030"]],AUTHORITY["EPSG","6326"]],PRIMEM["Greenwich",0],UNIT["degree",0.0174532925199433,AUTHORITY["EPSG","9122"]],AUTHORITY["EPSG","4326"]],PROJECTION["Albers_Conic_Equal_Area"],PARAMETER["latitude_of_center",23],PARAMETER["longitude_of_center",-96],PARAMETER["standard_parallel_1",29.5],PARAMETER["standard_parallel_2",45.5],PARAMETER["false_easting",0],PARAMETER["false_northing",0],UNIT["metre",1,AUTHORITY["EPSG","9001"]],AXIS["Easting",EAST],AXIS["Northing",NORTH]]'
+
+        crs = proj_to_cartopy(albers_wkt)
+
+        assert isinstance(crs, self.ccrs.AlbersEqualArea)
+        # Check that the projection parameters are correctly parsed
+        # Note: proj4_params might be available depending on cartopy version
+        if hasattr(crs, 'proj4_params'):
```

Comment:
I think we have this check based on cartopy version, so we don't silently let it be skipped if there is a rename.